daft/dataframe/display.py

```diff
@@ -21,27 +21,20 @@ def __init__(self, name: str, subgraph_id: str):
 
 class MermaidOptions:
     simple: bool
+    bottom_up: bool
     subgraph_options: Optional[SubgraphOptions]
 
-    def __init__(self, simple: bool = False, subgraph_options: Optional[SubgraphOptions] = None):
+    def __init__(self, simple: bool = False, bottom_up=False, subgraph_options: Optional[SubgraphOptions] = None):
         self.simple = simple
+        self.bottom_up = bottom_up
         self.subgraph_options = subgraph_options
 
     def with_subgraph_options(self, name: str, subgraph_id: str):
-        opts = MermaidOptions(self.simple, SubgraphOptions(name, subgraph_id))
+        opts = MermaidOptions(self.simple, subgraph_options=SubgraphOptions(name, subgraph_id))
 
         return opts
 
 
-def make_display_options(simple: bool, format: str) -> Union[MermaidOptions, AsciiOptions]:
```

Comment:
instead of using the rust enum, we're checking on the python side now, and calling the appropriate method. `repr_ascii` or `repr_mermaid`